src/composables/useBcrosNavigate.ts

```diff
@@ -41,10 +41,22 @@ export const useBcrosNavigate = () => {
   function goToSetupAccount () {
     redirect(config.public.authWebURL + 'setup-account')
   }
-  const goToDigitalCredentialsPage = () => {
+  function goToDigitalCredentialsPage () {
```

Comment:
What is the reason to converting it to this type of declaration, is it to keep it the same as other function definitions in the file?

Reply:
Yeah it is just to keep the declaration consistent within the file.